src/api/postFile.ts

```diff
@@ -2,28 +2,30 @@ import axios from 'axios';
 import { url } from '../shared/variable.shared';
 import { useUserStore } from 'src/stores/user';
 
-const changeFileToFormData = (file: File)=> {
-    const formData = new FormData();
-    formData.append('file', file, file.name);
-    return formData
-}
-
+const changeFileToFormData = (file: File) => {
+  const formData = new FormData();
+  formData.append('file', file, file.name);
+  return formData;
+};
 
 const sendFile = async (file: File): Promise<number> => {
-    const formData = changeFileToFormData(file);
-    const userStore = useUserStore();
-    const teamID = userStore.getUser.email && await userStore.loadTeam(userStore.getUser.email)
-    try{
-        const {data} = await axios.post(`${url}/projects/${teamID}`, formData, { headers: {
-            'Content-Type': `multipart/form-data`,
-            'Authorization': `Bearer${userStore.user.jwt}`
-        } })
-        return data.id;
-    }
-    catch(err: any) {
-        console.error(err);
-        return 0;
-    }
-}
+  const formData = changeFileToFormData(file);
+  const userStore = useUserStore();
+  const teamID =
+    userStore.getUser.email &&
+    (await userStore.loadTeam(userStore.getUser.email));
+  try {
+    const { data } = await axios.post(`${url}/projects/${teamID}`, formData, {
+      headers: {
+        'Content-Type': `multipart/form-data`,
```

Comment:
NIT: dałbym te multipart/form-data do jakieś stałej bo używamy to dla modeli, projektów i może gdzieś indziej użyjemy.

Reply:
Można kiedyś zrobić taska na clean code i ogarnąć wszystkie zmienne